dataprofiler/profilers/profiler_options.py

```diff
@@ -271,6 +273,9 @@ def __init__(self):
         self.skewness = BooleanOption(is_enabled=True)
         self.kurtosis = BooleanOption(is_enabled=True)
         self.histogram_and_quantiles = HistogramOption()
+        # By default, we do NOT want bias, hence bias is false. This
+        # convention is made to maintain consistency with scipy.stats
+        self.bias = BooleanOption(is_enabled=False)
```

Comment:
I think the naming might be a little confusing and it might be best to change it to `bias_correction`
as in `bias_correction.is_enabled = True` I like the idea of. it matching scipy, but since we utilize the is_enabled it is kinda weird to say we are enabling bias. Thoughts?

Reply:
Agreed, bias_correction.is_enabled is more semantically clear to me, will make those changes